user-interviews/ui-design-system
# Add default containerClassName to loading skeleton

[//]: # (
See https://www.notion.so/Expectations-for-Code-Reviews-4c261233caf64f12ab459ba090be2c50 for more best practices around how we do Code Review.
This is a default pull request template that teams can use as a starting point.
You can edit this here: docs/pull_request_template.md
)
### Related Issue(s)

Fix #1194

### Description
[//]: # (
Provide a brief description of the intent of this pull request.
Example: Allow users to search for a specific product by name.
)

Addressing https://www.npmjs.com/package/react-loading-skeleton#the-height-of-my-container-is-off-by-a-few-pixels

### Reviewers
[//]: # (
Indicate who your primary and secondary reviewers are, and make sure they know.
Do this AFTER you have done your own self-review.
)

Primary:  @kyleshike 

secondary: @jasonbasuil / @gabescarbrough  

cc: @user-interviews/rx-fast 

### Summary of Changes
[//]: # (
Provide a list of the major areas of change or screenshots and provide context that makes it easier to understand what is going on when first reading the code.
Think big picture, don't just replicate the *Files changed* tab or feel the need to have an item for each file changed if there are natural groupings, like adding a controller action and a route, or a view and a component.
The purpose is to call out anything you found confusing while developing, or that might not be obvious. What would you want a heads up about if you were the reviewer? If you were pairing what would you call out at the start of the call?
If there is something you're unsure about or need clarification on, then include your questions alongside the relevant topic.
)

Adding default `containerClassName` prop to `LoadingSkeleton` which sets `line-height: 1`


### Manual Test plan
[//]: # (
How did you test this manually? How can your reviewer test the workflow?
If this is a bug, how can you reproduce the bug?
)

- Start DS locally on this branch and verify that loading states with skeletons behave as expected (did that, but don't want to be including too many recordings since it's a public repo)
